package/post-install.sh

```diff
@@ -26,28 +26,49 @@ ddtrace.request_init_hook=${EXTENSION_AUTO_INSTRUMENTATION_FILE}
 EOF
 }
 
-function install_conf_d_file() {
+function install_conf_d_files() {
     INI_FILE_PATH="${EXTENSION_CFG_DIR}/$INI_FILE_NAME"
 
-    println "Creating ddtrace.ini"
-    println "\n"
+    if [[ ! -f $INI_FILE_PATH ]]; then
+        println "Creating ddtrace.ini"
+        println "\n"
 
-    append_configuration_to_file "${INI_FILE_PATH}"
+        append_configuration_to_file "${INI_FILE_PATH}"
 
-    println "ddtrace.ini created"
+        println "ddtrace.ini created"
+    else
+        println "Found an existing ddtrace.ini file"
+    fi
     println
 
-    PHP_DDTRACE_INI="$PHP_CFG_DIR/$INI_FILE_NAME"
+    println "Linking ddtrace.ini for all installed SAPI's"
+    println "\n"
+
+    # Detect installed SAPI's
+    SAPI_DIR=${PHP_CFG_DIR%cli/conf.d}
+    if [[ "$PHP_CFG_DIR" != "$SAPI_DIR" ]]; then
+        SAPI_CONFIG_DIRS=($(find ${SAPI_DIR} -type d -name 'conf.d'))
+    else
+        SAPI_CONFIG_DIRS=(${PHP_CFG_DIR})
+    fi
+
+    for SAPI_CFG_DIR in ${SAPI_CONFIG_DIRS[@]}
+    do
+        println "Found SAPI config directory: ${SAPI_CFG_DIR}"
 
-    println "Linking ddtrace.ini to ${PHP_DDTRACE_INI}"
-    test -f "${PHP_DDTRACE_INI}" && rm "${PHP_DDTRACE_INI}"
-    ln -s "$INI_FILE_PATH" "${PHP_DDTRACE_INI}"
+        PHP_DDTRACE_INI="$SAPI_CFG_DIR/$INI_FILE_NAME"
```

Comment:
this could be extracted to separate fuinciton allowing execution like:
```bash
ensure_file_linked $INI_FILE_PATH "$PHP_CFG_DIR/$INI_FILE_NAME"
ensure_file_linked $INI_FILE_PATH "$PHP_FPM_CFG_DIR/$INI_FILE_NAME"
```
etc.